client/python/conductor/conductor.py

```diff
@@ -219,7 +219,7 @@ def ackTask(self, taskId, workerid):
         params['workerid'] = workerid
         headers = {'Accept': 'application/json'}
         value = self.post(url, params, None, headers)
-        return value == 'true'
+        return value
```

Comment:
So, the Java client returns ack value as Boolean, which is explicit because of types. Whereas, the rest endpoint converts Boolean to String and returns that. In which case, the value would be amongst "true" or "false". Couple of concerns, changing value from boolean to string might break existing users. Do you have different expectations from this function? I.e do you expect it to return plain string, or json, or some other form?